src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -60,7 +60,41 @@ class PolicyHandlerInterface;
 namespace application_manager {
 namespace mobile_api = mobile_apis;
 
-/*
+/**
+ * @brief ResetGlobalPropertiesResult
+ * contains flags indicating success of
+ * ResetGlobalProperties operation.
+ * Used in MessageHelper functions
+ * to construct relevant requests
+ **/
+struct ResetGlobalPropertiesResult {
+  bool help_prompt;
+  bool timeout_prompt;
+  bool vr_help_title_items;
+  bool menu_name;
+  bool menu_icon;
+  bool keyboard_properties;
+  int number_of_reset_vr;
```

Comment:
i think this can be a bool too, the value is never read other than as bool

Reply:
@iCollin good point. Changed to bool in https://github.com/smartdevicelink/sdl_core/pull/3475/commits/0149f2ffce888d0985f5329a83791c48e55f9dbd